jx/scripts/ci.sh

```diff
@@ -54,7 +54,7 @@ git config --global --add user.email jenkins-x@googlegroups.com
 cp ${JX} /usr/bin
 
 # lets trigger the BDD tests in a new team and git provider
-${JX} step bdd -b --provider=gke --git-provider=ghe --git-provider-url=https://github.beescloud.com --git-username dev1 --git-api-token $GHE_CREDS_PSW --default-admin-password $JENKINS_CREDS_PSW --no-delete-app --no-delete-repo --tests install --tests test-create-spring
+${JX} step bdd -b --provider=gke --git-provider=ghe --git-provider-url=https://github.beescloud.com --git-username dev1 --git-api-token $GHE_CREDS_PSW --default-admin-password $JENKINS_CREDS_PSW --no-delete-app --no-delete-repo --tests install --tests test-create-spring --tests html-report
```

Comment:
It would be better, probably, to write a jx step that uses xunit-viewer to convert a junit report to html. In general we want to remove all the custom stuff from the Makefiles in the bdd tests

Reply:
I could install `xunit-viewer` and add the report generation after the jx bdd step like
```
$ jx bdd...
$ xunit-viewer --results=reports/junit.xml --output=reports/junit.html --title="BDD Tests"
```
I don't think there is value in adding a wrapper to the xunit-viewer?

Reply:
sounds good to me!